tutorials/server/ubuntu-desktop-aws/ubuntu-desktop-aws.md

```diff
@@ -0,0 +1,153 @@
+---
+id: tutorial-ubuntu-desktop-aws
+summary: Learn how to connect and run ubuntu desktop on AWS EC2 Instance. Access GUI to manage your machine more easily.
+categories: server
+tags: vnc,aws,ec2,tutorial,ubuntu,server,Ubuntu 16.04 LTS
+difficulty: 2
+status: published
+published: 2017-12-16
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+author: Simran Singh <invincible.simran@gmail.com>
+
+---
+
+# Running Ubuntu Desktop on AWS EC2 Instance
+
+## Overview
+Duration: 0:01
+
+This tutorial will guide you through the setup of running ubuntu desktop on AWS EC2 Instance using TightVNC on a
+system running Ubuntu 16.04+.
+
+What you'll need :
+
+* Access to an AWS EC2 instance using commandline - This tutorial assumes that you're logged into the machine using SSH.
+* A machine running ubuntu 16.04+.
+* Remmia Remote Desktop Client (Pre-installed on 16.04+)
+
+## Setting up TightVNC on AWS
+Duration: 0:02
+
+Let's install Ubuntu Desktop and TightVNC on your EC2 instance.
+After logging in to your EC2 instance using the terminal, enter the following commands to install the tools that will be required to run ubuntu desktop :
+
+Note: While installing VNC Server you'll be required to setup a password for the server.
```

Comment:
Tutorails use a different format for admonitions at the moment - use the syntax:
```
negative
: Note: While installing ...
```